### PR TITLE
Image seed

### LINKED ISSUE #5: Image Seeds
#### As a developer I want at least 20 images to be created when I setup the database.

Acceptance criteria:
- [x] After running `rake db:setup` at least 20 images are shown in the
  homepage.
- [x] When a new app is deployed on heroku (e.g., every review app), that app
  automatically has at least 20 images on its homepage.

Dependencies:
- Image Index

### PR DESCRIPTION
**Pull request by lsq921224**
Closes #5 
This branch is based on #21

### CODE REVIEW

File: db/seeds.rb [hunk 1]
```diff
@@ -5,3 +5,27 @@
...
+  http://oi68.tinypic.com/mbrbth.jpg
+  http://oi63.tinypic.com/28auo8h.jpg
+  http://oi65.tinypic.com/3a7nb.jpg
+  http://oi63.tinypic.com/dzgya0.jpg
+  http://oi64.tinypic.com/dzhtv8.jpg
+  http://oi66.tinypic.com/dzhyiv.jpg
+  http://oi65.tinypic.com/dzi5pl.jpg
+).freeze
+
+Image.create!([SAMPLE_LINKS.map { |url| { link: url } }])
```
Comment (bboe) [inline db/seeds.rb]: ⭐  Awesome

File: app.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "ropes",
   "scripts": {
-    "postdeploy": "rails db:migrate"
+    "postdeploy": "rails db:migrate db:seed"
```
Comment (bboe) [inline app.json]: Did you try `rails db:setup` again? Running migrations should not need to happen in the presence of a working schema.rb file.

If `rails db:setup` doesn't work, please try instead `rails db:schema:load db:seed`

**Comment by bboe** [comment]
Just one comment pertaining to preferring schema.rb over migrations.

**Comment by lsq921224** [comment]
db:setup still doesn't work, db:schema:load works well!

**Comment by bboe** [comment]
Great. Looks good. Please squash, force push, verify there is only one commit on the PR and then merge. Let me know if this process gives you trouble.

**Comment by bboe** [comment]
image

Please delete your branch once merged.

**Pull request merged**